packages/grid/data-grid/src/tests/filtering.DataGrid.test.tsx

```diff
@@ -0,0 +1,125 @@
+import * as React from 'react';
+import { createClientRenderStrictMode } from 'test/utils/index';
```

Comment:
```suggestion
import { createClientRenderStrictMode } from 'test/utils';
```

x all the other places